File: IT/Funzioni/Esercizi/Python/esercizio7.py
# file esercizio7.py

from __future__ import print_function

def maximum(a, b):
	return a if a > b else b

a = 5
b = 10
c = 15

print("Massimo tra {} e {}: {}".format(a, b, maximum(a, b)))
print("Massimo tra {} e {}: {}".format(b, c, maximum(b, c)))

File: IT/Funzioni/Esercizi/Java/Esercizio7.java
/* file Esercizio7.java */

public class Esercizio7 {
	static int maximum(int a, int b) {
		return a > b ? a : b;
	}

	public static void main(String [] args) {
		int a = 5;
		int b = 10;
		int c = 15;

		System.out.println("Massimo tra " + a + " e " + b + ": " + maximum(a, b));
		System.out.println("Massimo tra " + b + " e " + c + ": " + maximum(b, c));
	}
}

